skeleton/pom.xml

```diff
@@ -90,7 +90,6 @@
         <dependency>
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-server</artifactId>
-            <scope>test</scope>
```

Comment:
This package contains the GuiceContainer class, which is used for the exception mapping.
See: https://github.com/killbill/killbill-commons/pull/16/files#diff-a2c6a89915677e225f2f5b804057314bR46